radicle-httpd/src/api/json.rs

```diff
@@ -117,6 +119,7 @@ pub(crate) fn patch(id: PatchId, patch: Patch) -> Value {
                 "description": rev.description(),
                 "base": rev.base,
                 "oid": rev.oid,
+                "branches": get_branches(repo, patch.author().id(), &rev.oid).unwrap_or(vec![]),
```

Comment:
So one important detail is that these may not all be branches. I think tags could be returned as well here, or refs that aren't branches.

I'd suggest calling this key `refs` for now, to be correct. Later, we could filter it to only include branches, but it's a bit trickier.

Reply:
Also would change the function to `get_refs`